### PR TITLE
feature/se-5-handle-date

### LINKED ISSUE #4: feature/se-5-handle-unknown-date
- [x] If local.approximateDate.issued metadata is not null (exist) load its value to the dc.date.issued
- [x] Create test: If local.approximateDate.issued is not null, dc.date.issued value should be overwritten by local.approximateDate.issued value.

### PR DESCRIPTION
**Pull request by milanmajchrak**
| Phases            | MM  |  MB  | MR |   JM | Total  |
|-----------------|----:|----:|-----:|-----:|-------:|
| ETA                  |  4.8  |    0 |     0 |      0 |        0 |
| Developing      |  12+1  |    0 |    0 |      0 |         0 |
| Review             |  0  |    0 |    0 |      0 |         0 |
| Total                |   -  |   -   |  -    |   -    |         0 |
| ETA est.             |      |       |       |         |         0 |
| ETA cust.           |   -  |   -  |   -   |   -     |        0 |
## Problem description
Date of creation of some items has unknown or in date range. This information is stored in the approximateDate metadata. If approximateDate metadata is not null show value from this metadata.

## Problems
This solution doesn't seems to be right because I changed RemoteObject which is constant.
**Solution:** Solve this problem in the backend: https://github.com/dataquest-dev/DSpace/pull/52

### CODE REVIEW

File: src/app/item-page/item-page.resolver.ts [hunk 1]
```diff
@@ -43,7 +44,13 @@ export class ItemPageResolver extends ItemResolver {
             this.router.navigateByUrl(itemRoute + subRoute);
           }
         }
-        return rd;
+
```
Comment (vidiecan) [inline src/app/item-page/item-page.resolver.ts:47]: why?

**Review by vidiecan — CHANGES_REQUESTED** [review]

File: src/app/item-page/item-page.resolver.ts [hunk 2]
```diff
@@ -43,7 +44,13 @@ export class ItemPageResolver extends ItemResolver {
...
+        const copiedRd = _.cloneDeep(rd);
```
Comment (milanmajchrak) [inline src/app/item-page/item-page.resolver.ts:48]: @vidiecan I found used _cloneDeep_ in the DSpace7: https://github.com/DSpace/dspace-angular/blob/main/src/app/item-page/edit-item-page/item-metadata/item-metadata.component.ts#L129

Comment (milanmajchrak) [inline src/app/item-page/item-page.resolver.ts:48]: @vidiecan I found used _cloneDeep_ in the DSpace7: https://github.com/DSpace/dspace-angular/blob/main/src/app/item-page/edit-item-page/item-metadata/item-metadata.component.ts#L129

Comment (milanmajchrak) [reply src/app/item-page/item-page.resolver.ts:48]: maybe cloneDeep is not so bad idea

Comment (milanmajchrak) [reply src/app/item-page/item-page.resolver.ts:48]: maybe using cloneDeep is not so bad idea

**Pull request closed without merging**